src/frontend/src/lib/components/convert/ConvertAmountSource.svelte

```diff
@@ -82,16 +103,13 @@
 
 	<button
 		slot="balance"
-		class={`rounded px-2 py-0.5 ${isZeroBalance ? 'bg-error-subtle-alt text-error' : 'bg-brand-subtle text-brand-primary'}`}
+		class={`rounded px-2 py-0.5 ${isZeroBalance ? 'bg-error-subtle-alt text-error' : isNullish(maxAmount) ? 'animate-pulse bg-disabled text-tertiary' : 'bg-brand-subtle text-brand-primary'}`}
```

Comment:
When the text changes, is the button transition animated? If not, you might want to `transition-all` or something like that (don't remember exactly the tailwind class for it) to smoothen the transitions

Reply:
good idea, added it